docs/src/refs.bib

```diff
@@ -0,0 +1,611 @@
+  title    = {On the problem of penetration in particle methods},
+  year     = {1989},
+  issn     = {0021-9991},
+  number   = {1},
+  pages    = {1-15},
+  volume   = {82},
+  abstract = {A method is described which prevents penetration when particle methods are used to simulate streams of fluid impinging on each other. The method does not produce dissipation but it does produce extra dispersion.},
+  doi      = {10.1016/0021-9991(89)90032-6},
+  url      = {https://www.sciencedirect.com/science/article/pii/0021999189900326},
+}
+
+
+@Article{Fourtakas2019,
+  author    = {Fourtakas, Georgios and Dominguez, Jose M. and Vacondio, Renato and Rogers, Benedict D.},
+  journal   = {Computers and; Fluids},
+  title     = {Local uniform stencil (LUST) boundary condition for arbitrary 3-D boundaries in parallel smoothed particle hydrodynamics (SPH) models},
+  year      = {2019},
+  issn      = {0045-7930},
+  month     = aug,
+  pages     = {346--361},
+  volume    = {190},
+  doi       = {10.1016/j.compfluid.2019.06.009},
+  publisher = {Elsevier BV},
+}
+
+
+@Article{Molteni2009,
+  author    = {Molteni, Diego and Colagrossi, Andrea},
+  journal   = {Computer Physics Communications},
+  title     = {A simple procedure to improve the pressure evaluation in hydrodynamic context using the SPH},
+  year      = {2009},
+  issn      = {0010-4655},
+  month     = jun,
+  number    = {6},
+  pages     = {861--872},
+  volume    = {180},
+  doi       = {10.1016/j.cpc.2008.12.004},
+  publisher = {Elsevier BV},
+}
+
+
+@Article{Ferrari2009,
+  author    = {Ferrari, Angela and Dumbser, Michael and Toro, Eleuterio F. and Armanini, Aronne},
+  journal   = {Computers and; Fluids},
+  title     = {A new 3D parallel SPH scheme for free surface flows},
+  year      = {2009},
+  issn      = {0045-7930},
+  month     = jun,
+  number    = {6},
+  pages     = {1203--1217},
+  volume    = {38},
+  doi       = {10.1016/j.compfluid.2008.11.012},
+  publisher = {Elsevier BV},
+}
+
+
+@Article{Antuono2010,
+  author    = {Antuono, M. and Colagrossi, A. and Marrone, S. and Molteni, D.},
+  journal   = {Computer Physics Communications},
+  title     = {Free-surface flows solved by means of SPH schemes with numerical diffusive terms},
+  year      = {2010},
+  issn      = {0010-4655},
+  month     = mar,
+  number    = {3},
+  pages     = {532--549},
+  volume    = {181},
+  doi       = {10.1016/j.cpc.2009.11.002},
+  publisher = {Elsevier BV},
+}
+
+@Article{Crespo2007,
+  author  = {Crespo, Alejandro and Gesteira, Moncho and Dalrymple, Robert},
+  journal = {CMC. Computers, Materials, and Continua},
+  title   = {Boundary conditions generated by dynamic particles in SPH methods},
+  year    = {2007},
+  month   = {06},
+  volume  = {3},
+}
+
+@Article{Cole2009,
+  author  = {Cole, Robert},
+  journal = {Physics Today},
+  title   = {Underwater Explosions},
+  year    = {2009},
+  month   = {01},
+  pages   = {35-35},
+  volume  = {1},
+  doi     = {10.1063/1.3066176},
+}
```

Comment:
```suggestion
@book{Cole1948,
  location = {Princeton, New Jersey},
  title = {Underwater Explosions},
  publisher = {Princeton University Press},
  author = {{Robert Hugh Cole}},
  date = {1948},
}
```
What happened here? How did the year change? Have you checked that the other bibitems are correct?

Reply:
Idk. Maybe the auto generator used the year where the work was digitized. 

This is the eprint version:
``` 
@Article{Cole1948,
  author    = {Cole, Robert H. and Weller, Royal},
  journal   = {Physics Today},
  title     = {Underwater Explosions},
  year      = {1948},
  issn      = {1945-0699},
  month     = oct,
  number    = {6},
  pages     = {35--35},
  volume    = {1},
  doi       = {10.1063/1.3066176},
  publisher = {AIP Publishing},
}
```